pkg/local_object_storage/shard/get_test.go

```diff
@@ -68,6 +63,10 @@ func testShardGet(t *testing.T, sh *shard.Shard) {
 	})
 
 	t.Run("parent object", func(t *testing.T) {
+		if hasWriteCache {
+			t.Skip()
```

Comment:
I think we still can pass this and `TestShard_Head/with_write_cache/virtual_object` test without skip.

The issue with these tests is that write cache do not index objects in metabase immediately. However indexing happens when data flushed to persisted storage (disk cache or even blobstor). If it is possible to wait for flush or invoke it somehow, I would do this:

```go
if hasWriteCache {
	_, err = sh.Get(getPrm)
	require.True(t, errors.Is(err, object.ErrNotFound))
	// invoke flush to persistent storage somehow, sleep maybe?
}
_, err = sh.Get(getPrm)
var expectedErr *objectSDK.SplitInfoError
require.True(t, errors.As(err, &expectedErr))
si, ok := err.(*objectSDK.SplitInfoError)
...
```